modules/light-clients/08-wasm/keeper/msg_server_test.go

```diff
@@ -88,11 +97,146 @@ func (suite *KeeperTestSuite) TestMsgStoreCode() {
 					suite.Require().Contains(events, evt)
 				}
 			} else {
-				suite.Require().Error(err)
 				suite.Require().ErrorIs(err, tc.expError)
 				suite.Require().Nil(res)
 				suite.Require().Empty(events)
 			}
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestMsgMigrateContract() {
+	oldCodeHash := sha256.Sum256(wasmtesting.Code)
+
+	mockByteCode := []byte("MockByteCode-TestMsgMigrateContract")
```

Comment:
ditto `wwasmtesting.Code`

Reply:
`wasmtesting.Code` is the oldCode, so the `mockByteCode` is the new Code

Reply:
ah, maybe a rename then to indicate that

Reply:
done